buildprocess.tex

```diff
@@ -4,6 +4,69 @@
 
 \indextext{build|(}%
 
+\rSec1[build.import]{Import relationship between translation units}
+
+\pnum The name of a module used in the ``import'' and ``module''
+keywords is the ``logical name'' of a module.
+
+\pnum When a translation unit imports a named module, the translation
+unit doing the import has an import dependency on the translation unit
+providing the importable unit identified via the ``logical name''.
+
+\pnum The translation of an importable unit is externalized into an
+intermediate artifact, the Built Module Interface file (BMI). That
+file is then consumed when importing a named module.
+
+\rSec1[build.arguments]{Compiler arguments and the import relationship}
+
+\pnum ``ABI coherency'' is the expected level of coherency in the
+compilation command for the translation of the code in the purview of a
+module and all translation units importing that module. When that
+level of coherency is not met, the final program may fail to link or
+it may link and have undefined behavior at runtime.
+
+\pnum ``BMI coherency'' is the expected level of coherency in the
+compiler arguments for a BMI file to be used when importing a
```

Comment:
```suggestion
\rSec1[build.arguments]{Compilation commands and the import relationship}

\pnum ``ABI coherency'' is the expected level of coherency in the
compilation command for the translation of the code in the purview of a
module and all translation units importing that module. When that
level of coherency is not met, the final program may fail to link or
it may link and have undefined behavior at runtime.

\pnum ``BMI coherency'' is the expected level of coherency in the
compiler arguments for a BMI file to be used when importing a
```